content/docs/2.12/scalers/apache-kafka.md

```diff
@@ -418,3 +421,69 @@ spec:
     authenticationRef:
       name: keda-trigger-auth-kafka-credential
 ```
+
+#### Your kafka cluster turns on SASL/GSSAPI auth without TLS:
+
+##### `sasl/gssapi` in TriggerAuthentication
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: keda-kafka-secrets
+  namespace: default
+data:
+  sasl: "gssapi"
+  tls: "disable"
+  username: "admin"
+  realm: <your kerberos realm>
+  keytab: <your kerberos keytab>
+  kerberosConfig: <your kerberos configuration>
```

Comment:
This should really be coming from a ConfigMap rather than a secret.  Same for a number of values above.  Except for the `keytab` nothing is actually secret here and applications often already have this defined in ConfigMap's